misc/drone-scripts/setup.sh

```diff
@@ -19,22 +19,29 @@ govc snapshot.revert -vm $ESX_6_0 init
 
 echo "Waiting for revert to complete";
 
+DIR=$(dirname ${BASH_SOURCE[0]})
+. $DIR/../scripts/wait_for.sh
+
+# Threshold to time out 
+retryCount=30
+
 echo ESX 6.5
-until govc vm.ip $ESX_6_5
-do
-    echo "Waiting for revert to complete for ESX6.5";
-    sleep 1;
-done
+wait_for "govc vm.ip $ESX_6_5" $retryCount
 
 echo ESX 6.0
-until govc vm.ip $ESX_6_0
-do
-    echo "Waiting for revert to complete for ESX6.0";
-    sleep 1;
-done
+wait_for "govc vm.ip $ESX_6_0" $retryCount
 
 echo "Reset complete"
-sleep 5;
+
+export GOVC_URL=$GOVC_URL_6_5
+export GOVC_USERNAME=$GOVC_USERNAME_ESX
+export GOVC_PASSWORD=$GOVC_PASSWORD_ESX
+
+echo "Wait for VM to get ready"
+
+wait_for "$GOVC_GET_IP photon.vmfs" $retryCount
+wait_for "$GOVC_GET_IP Ubuntu.16.10" $retryCount
+
```

Comment:
Sure, I will add some inline comments for the future reference. Thanks for pointing out!

> Or loop over a set of VMs and do the wait in sequence.

Could do that. Based on my experience, the issue has been observed against ESX 6.5 as it is the first one to run tests against so it would be OK for user not to add any more vms to the list unless replacing them with new one.